dtfs-central-api/src/v1/controllers/utilisation-report-service/post-remove-fees-from-payment.controller/helpers.ts

```diff
@@ -0,0 +1,44 @@
+import { FeeRecordEntity, UtilisationReportEntity } from '@ukef/dtfs2-common';
+import { TfmSessionUser } from '../../../../types/tfm/tfm-session-user';
+import { executeWithSqlTransaction } from '../../../../helpers';
+import { UtilisationReportStateMachine } from '../../../../services/state-machines/utilisation-report/utilisation-report.state-machine';
+import { InvalidPayloadError } from '../../../../errors';
+
+export const validateAtLeastOneFeeRecordSelected = (selectedFeeRecordIds: number[]) => {
+  if (selectedFeeRecordIds.length === 0) {
+    throw new InvalidPayloadError('No fee records selected.');
+  }
+};
+
+export const validateNotAllFeeRecordsSelected = (selectedFeeRecordIds: number[], totalSelectableFeeRecords: number) => {
+  if (selectedFeeRecordIds.length === totalSelectableFeeRecords) {
```

Comment:
What if the length is greater?

Reply:
Added a separate validation function for this case - changes in commit fa2ed806af7724815b27bc4f2a04dabf93dd1a78